Library/Homebrew/extend/os/linux/system_config.rb

```diff
@@ -14,6 +15,18 @@ def host_os_version
       end
     end
 
+    def maybe_na(version)
```

Comment:
Is this method going to be used in other PRs? Otherwise can you just roll it into `host_glibc_version` with a `return "N/A" if version.null?`

Reply:
Done.